spaces/urls.py

```diff
@@ -20,4 +21,5 @@
     path("spaces/<int:pk>/members/", space_members, name="space_members"),
     path("spaces/<int:pk>/join/", join_space, name="join_space"),
     path("spaces/<int:pk>/leave/", leave_space, name="leave_space"),
+    path("newspace/", newspace, name="newspace"),
```

Comment:
@MariaAsif135 I am moving this to spaces app since it is related with that.

Reply:
Thanks, we need to work on the integration part of this newspace part with the space join functions.